example/swagger-files/petstore.json

```diff
@@ -222,7 +222,8 @@
           "in": "formData",
           "description": "Updated name of the pet",
           "required": false,
-          "type": "string"
+          "type": "string",
```

Comment:
Can you instead add this here: https://github.com/readmeio/api-explorer/blob/master/example/swagger-files/types.json#L57-L60?